hwdb.d/60-sensor.hwdb

```diff
@@ -315,6 +315,11 @@ sensor:modalias:platform:cros-ec-accel:dmi:*:svnGoogle:pnCaroline*
 # Dell Inspiron Chromebook 14 2-in-1
 sensor:modalias:platform:cros-ec-accel:dmi:*svnGoogle:pnVayne*
  ACCEL_MOUNT_MATRIX=-1, 0, 0; 0, -1, 0; 0, 0, -1
+ 
```

Comment:
There's a trailing whitespace  on this line which is causing the CI fails.

Reply:
Oops! Deleted!

Reply:
The trailing newline you dropped was alright, it's the trailing whitespace on the first line of your patch which is causing all this havoc :-)

![image](https://user-images.githubusercontent.com/679338/86643007-dde10780-bfcb-11ea-9b29-57b45558e505.png)

Reply:
Thanks for the assist. Trying this again!